### PR TITLE
add filter data endpoint

### PR DESCRIPTION
**Pull request by SvetaMemphis**

### CODE REVIEW

File: models/user_mgmt.go [hunk 1]
```diff
@@ -77,3 +77,12 @@ type EditAvatarSchema struct {
 type EditAnalyticsSchema struct {
 	SendAnalytics bool `json:"send_analytics"`
 }
+
+type GetFilterDetailsSchema struct {
+	Route string  `form:"route" json:"route"`
+
```
Comment (idanasulinmemphis) [inline models/user_mgmt.go]: remove empty line

File: http_server/routes/user_mgmt.go [hunk 2]
```diff
@@ -38,4 +38,5 @@ func InitializeUserMgmtRoutes(router *gin.RouterGroup) {
 	userMgmtRoutes.GET("/getCompanyLogo", userMgmtHandler.GetCompanyLogo)
 	userMgmtRoutes.PUT("/editAnalytics", userMgmtHandler.EditAnalytics)
 	userMgmtRoutes.POST("/skipGetStarted", userMgmtHandler.SkipGetStarted)
+	userMgmtRoutes.GET("/getFilterDetails", userMgmtHandler.GetFilterDetails)
```
Comment (shohamroditimemphis) [inline http_server/routes/user_mgmt.go:41]: do command + shift +  f to prettier the code

File: server/memphis_handlers_user_mgmt.go [hunk 3]
```diff
@@ -1090,3 +1090,102 @@ func (umh UserMgmtHandler) SkipGetStarted(c *gin.Context) {
...
+			Name:  tag.Name,
+			Color: tag.Color,
+		}
+		tagsRes = append(tagsRes, tagRes)
+	}
+	return tagsRes, err
+}
+
+func (umh UserMgmtHandler) GetFilterDetails (c *gin.Context){
+
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_user_mgmt.go:1136]: remove empty line

File: server/memphis_handlers_user_mgmt.go [hunk 4]
```diff
@@ -1090,3 +1090,102 @@ func (umh UserMgmtHandler) SkipGetStarted(c *gin.Context) {
...
+		tags, err := umh.GetActiveTags()
+		if err != nil {
+			serv.Errorf("GetActiveTags error: " + err.Error())
+			c.AbortWithStatusJSON(500, gin.H{"message": "Server error"})
+			return
+		}
+
+		storage := []string {"memory", "disk"}
+
+		c.IndentedJSON(200, gin.H{"tags":tags, "users": users, "storage":storage})
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_user_mgmt.go]: add empty return

File: server/memphis_handlers_user_mgmt.go [hunk 5]
```diff
@@ -1090,3 +1090,102 @@ func (umh UserMgmtHandler) SkipGetStarted(c *gin.Context) {
...
+		if err != nil {
+			serv.Errorf("GetActiveTags error: " + err.Error())
+			c.AbortWithStatusJSON(500, gin.H{"message": "Server error"})
+			return
+		}
+
+		schemaType := []string {"protobuf", "json"}
+		usage := []string {"used", "not used"}
+
+		c.IndentedJSON(200, gin.H{"tags":tags, "users": users, "type":schemaType, "usage":usage})
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_user_mgmt.go]: add empty return

File: server/memphis_handlers_user_mgmt.go [hunk 6]
```diff
@@ -1090,3 +1090,102 @@ func (umh UserMgmtHandler) SkipGetStarted(c *gin.Context) {
...
+	case "syslogs":
+		logType := []string {"error","warn","info"}
+
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_user_mgmt.go:1182]: remove empty line

File: server/memphis_handlers_user_mgmt.go [hunk 7]
```diff
@@ -1090,3 +1090,102 @@ func (umh UserMgmtHandler) SkipGetStarted(c *gin.Context) {
...
+			serv.Errorf("GetActiveTags error: " + err.Error())
+			c.AbortWithStatusJSON(500, gin.H{"message": "Server error"})
+			return
+		}
+
+		storage := []string {"memory", "disk"}
+
+		c.IndentedJSON(200, gin.H{"tags":tags, "users": users, "storage":storage})
+	case "schemaverse":
+        users, err := umh.GetActiveUsers()
```
Comment (shohamroditimemphis) [inline server/memphis_handlers_user_mgmt.go]: move the lines in cases to func

File: server/memphis_handlers_user_mgmt.go [hunk 8]
```diff
@@ -1090,3 +1090,102 @@ func (umh UserMgmtHandler) SkipGetStarted(c *gin.Context) {
...
+	case "syslogs":
+		logType := []string {"error","warn","info"}
+
+		c.IndentedJSON(200, gin.H{"type":logType})
+
+	default:
+		c.IndentedJSON(200, gin.H{})
+
+	}
+
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_user_mgmt.go]: remove empty lines

**Pull request merged**